### ISSUE TITLE
RX 6900 XTX Chip

### ISSUE THREAD

**Issue by AyOhEff** [todxx/teamredminer#584]
Hi Team,

Your miner, like all other miners, didn't support newest 6900 XTX chip. I have this cards in reference design by AMD, and they have next configuration in stock:

AMDMemInfo for Hive OS v2.1.20
original code by Zuikkis and Yann St.Arnaud

-----------------------------------
Found Card: 1002:73af rev c0 (AMD Radeon RX 6900 XTX)
Chip Type: Sienna_Cichlid
BIOS Version: 113-D4120600-100
PCI: 0e:00.0
OpenCL Platform: 0
OpenCL ID: 2
Subvendor:  0x1002
Subdevice:  0x0e3a
Sysfs Path: /sys/bus/pci/devices/0000:0e:00.0
Memory Type: GDDR6
Memory Model: Samsung GDDR6
-----------------------------------

amd-info

=== GPU 0, 03:00.0 Radeon RX 6900 XTX 16368 MB ===
  Bios: 113-D4120600-100, ROM flash: Supported
  Core: 1400 MHz 637mV, Mem: 1155 MHz
  PerfCtrl: manual, Load: 99%, MemLoad: 100%, Power: 126.0 W, Cap: 284 W
  Core: 48°C, HotSpot: 54°C, Mem: 62°C, Fan: 0%, RPM: 0
  Core state: 1, clocks: 500 1400*
  Mem  state: 3, clocks: 96 456 673 1155*
  SOC  state: 1, clocks: 872 1200*
  DCEF state: 1, clocks: 417 960* 1200
  F    state: 1, clocks: 1551 1941*
  PCIE Link speed: n/a, PCIE Link width: n/a
  Memory total: 16368.00 MB, used: 5129.29 MB, free: 11238.71 MB, type: Samsung GDDR6
===

The main problem is in Mh/s - only 53, I can put memory clock 1200+, and cards working well but max. MH/s what I took is 55.2.

Could you please explain me why mh/s is too low for this cads?

WBR.

**Comment by Lumattix** [comment]
Finally, someone with the same problem as me.  No matter what you do, the reference AMD 6900XTX will not hash over 54Mh/s see my post here > https://www.reddit.com/r/EtherMining/comments/t40ifh/amd_reference_6900_xtx_lc_low_hash_rate_no_matter/

I asked for a fix in the Discord and everyone just gives me normal 6900XT clocks etc and doesn't seem to understand that they're different.

**Comment by AyOhEff** [comment]
@todxx, I hope that with the release of new maps you will pay attention to our problem.

My card is a second on a screenshot RX 6900 X TLC and it have the same parameters as RX6950 XT:
`https://www.overclockers.ua/news/video/131174-amd-rx6950-rx6750-rx6650-5.png`

**Comment by Lumattix** [comment]
@todxx 

I agree with the above, my card is also the 6900 XTX LC and specs look identical to new 6000 releases and our cards don't seem to have any optimization. Will new 6000 series be fine tuned?

**Comment by CamJava007** [comment]
I don't think this is true... as I have the latest 6900 xtx and have never had an issue.

**Comment by Lumattix** [comment]
> I don't think this is true... as I have the latest 6900 xtx and have never had an issue.

When you say the "latest" - Please elaborate which card. 

The card we're having issues with are these > https://videocardz.com/newz/sapphire-radeon-rx-6900-xt-lc-listed-is-this-the-unreleased-rx-6900-xtx

R-Mode, latest drivers, Windows or Hive. No matter what is done, they don't hash more than 53Mh/s

**Comment by CamJava007** [comment]
Powercolor Red Devil RX 6900 XTU with the XTXH chip.

**Comment by Lumattix** [comment]
@AyOhEff - I flashed the Asus 6900 XT LC bios to my card and getting around 58.5Mh/s now while still tuning settings

It seems the new 18Gbps memory is the same as 16Gbps but slower latency causing the issue to not mine faster

So flashing to older 16Gbps having faster latency helps

**Comment by AyOhEff** [comment]
> @AyOhEff - I flashed the Asus 6900 XT LC bios to my card and getting around 58.5Mh/s now while still tuning settings
> 
> It seems the new 18Gbps memory is the same as 16Gbps but slower latency causing the issue to not mine faster
> 
> So flashing to older 16Gbps having faster latency helps

Hi! Try to flash new reference design BIOS [AMD RX 6950 XT 16 GB BIOS](https://www.techpowerup.com/vgabios/245777/amd-rx6950xt-16384-220310).

**Comment by Lumattix** [comment]
@AyOhEff have you actually tried this and it works? how did you flash? under linux/hive?

**Comment by AyOhEff** [comment]
@Lumattix, in many attempt to took higher MH/s from this card i took better perfomance on [Asus RX 6900 XT 16 GB BIOS
(STRIX LC)](https://www.techpowerup.com/vgabios/230880/asus-rx6900xt-16384-201202), flashed from HiveOS - Core 1200 / Mem 1050 - now i have 62 MH/s.

**Comment by Lumattix** [comment]
> @Lumattix, in many attempt to took higher MH/s from this card i find better perfomance on [Asus RX 6900 XT 16 GB BIOS (STRIX LC)](https://www.techpowerup.com/vgabios/230880/asus-rx6900xt-16384-201202), flashed from HiveOS - Core 1200 / Mem 1050 - now i have 62 MH/s.

Thanks! Wish we could get higher but 62 will have to do

**Comment by Lumattix** [comment]
> @Lumattix, in many attempt to took higher MH/s from this card i find better perfomance on [Asus RX 6900 XT 16 GB BIOS (STRIX LC)](https://www.techpowerup.com/vgabios/230880/asus-rx6900xt-16384-201202), flashed from HiveOS - Core 1200 / Mem 1050 - now i have 62 MH/s.

Hey @AyOhEff 

Do you know where I can get stock bios for the AMD 6900XT LC cause I want to flash back but can't find where I put the original bios

**Comment by AyOhEff** [comment]
> 

@Lumattix upload to my Cloud Drive: [RX 6900 XTX](https://cloud.mail.ru/public/oaUM/TX9jTP1a9)

**Comment by Lumattix** [comment]
@AyOhEff are you still around?

Any chance you could upload the original LC bios again? The link above is dead
